sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/Employee.java

```diff
@@ -0,0 +1,41 @@
+package com.exadel.sandbox.team5.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+
+@Entity
+@Table(name = "employee")
+public class Employee {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO, generator = "native")
+    @Column(name = "id")
+    private Long id;
```

Comment:
Let's create `BaseEntity` with field `id` and use it as parent for such entity classes

Reply:
What is annotations I should use  with this class? all Lombok annotations and @Entity?

Reply:
We can use `@MappedSuperclass` and make this class as abstract. Also some Lombok annotations `@Data`.